src/common/config/src/lib.rs

```diff
@@ -20,6 +20,8 @@ use serde::{Deserialize, Serialize};
 #[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq)]
 #[serde(default)]
 pub struct WalConfig {
+    // wal directory
+    pub dir: Option<String>,
```

Comment:
Can you please update related config examples under `config/` to reflect this change?

Reply:
I found that it's already in `datanode.example.toml`, it was not removed before:

https://github.com/GreptimeTeam/greptimedb/blob/59d4081f7aed62ede33097abf526827f2e6eabc6/config/datanode.example.toml#L35

So we can just add it into `standalone.example.toml` wal section too.